src/utils/numbers.js

```diff
@@ -46,3 +47,7 @@ export const calculateMinPayment = (interest) => {
   const interestWithMargin = interest.mul(10010).div(10000);
   return interestWithMargin.lt(floor) ? floor : interestWithMargin;
 };
+
+export const calculateInterestRate = (borrowRatePerBlock, chainId) => {
+  return format((borrowRatePerBlock * BlocksPerYear[chainId]).toString());
```

Comment:
Use BigNumbers always 🖖 `borrowRatePerBlock.mul(BlocksPerYear[chainId])` and then update the usage to pass in BigNumber